src/main/java/baritone/utils/ToolSet.java

```diff
@@ -87,12 +88,9 @@ public double getStrVsBlock(BlockState state) {
      * @return values from 0 up
      */
     private int getMaterialCost(ItemStack itemStack) {
-        if (itemStack.getItem() instanceof TieredItem) {
-            TieredItem tool = (TieredItem) itemStack.getItem();
-            return (int) tool.getTier().getAttackDamageBonus();
-        } else {
-            return -1;
-        }
+        Tool toolComponent = itemStack.get(DataComponents.TOOL);
+        if (toolComponent == null) return -1;
+        return toolComponent.damagePerBlock(); // todo: i have no idea what "material cost" means anymore
```

Comment:
It's supposed to be a rough estimate how valuable the tool is. Just so if you happen to have two equally fast tools, Baritone will pick the cheaper one.

Reply:
gotcha, that makes more sense.

i think there's an enchantment cost data component for something like the prev behavior then. i'll update and also add clarifying comment about what this "cost" means